spannerlib/wrappers/spannerlib-ruby/lib/spannerlib/ffi.rb

```diff
@@ -0,0 +1,224 @@
+require 'rubygems'
+require 'bundler/setup'
+
+require 'google/protobuf'
+require 'google/rpc/status_pb'
+
+require 'ffi'
+
+module SpannerLib
+  extend FFI::Library
+  ffi_lib File.expand_path('../../shared/spannerlib.so', __dir__)
```

Comment:
This is fine for now, but eventually you need to find a strategy for where the binary is stored in the Ruby gem, and how you load the right os/arch version of that binary.